packages/manager/modules/exchange/src/account/update/account-update.html

```diff
@@ -1,79 +1,30 @@
-                role="alert"
-                data-ng-class="alertType"
-                data-ng-if="ctrl.noDomainMessage"
-            >
-                <div>
-                    <span
-                        class="fa fa-times float-right"
-                        data-ng-click="resetAction()"
-                    ></span>
-                    <span data-ng-bind-html="ctrl.noDomainMessage"></span>
-                </div>
-            </div>
+            class="mb-3"
+            data-ng-bind=":: 'exchange_account_update_header' | translate"
+        ></div>
 
-            <form
-                name="ctrl.updateAccountForm"
-                data-ng-if="!ctrl.noDomainMessage"
-            >
-                <div
-                    class="form-group"
-                    data-ng-class="{'has-error': ctrl.updateAccountForm.selectedAccountAddress.$dirty && (ctrl.updateAccountForm.selectedAccountAddress.$invalid || ctrl.takenEmailError)}"
-                >
-                    <label
-                        class="control-label required"
-                        for="selectedAccountAddress"
-                        data-translate="exchange_ACTION_update_account_step1_email_label"
-                    ></label>
-                    <div class="input-group">
-                        <input
-                            type="text"
-                            class="form-control"
-                            id="selectedAccountAddress"
-                            maxlength="256"
-                            name="selectedAccountAddress"
-                            required
-                            data-ng-change="ctrl.checkTakenEmails()"
-                            data-ng-disabled="!ctrl.canChangePrimary()"
-                            data-ng-model="ctrl.selectedAccount.login"
-                            data-ng-pattern="/^[-_a-zA-Z0-9]+((\.|\+)[-_a-zA-Z0-9]+)*$/"
-                        />
-                        <span class="input-group-addon">@</span>
-                        <select
-                            class="form-control"
-                            required
-                            data-ng-change="ctrl.checkTakenEmails()"
-                            data-ng-disabled="!ctrl.canChangePrimary()"
-                            data-ng-model="ctrl.selectedAccount.completeDomain"
-                            data-ng-options="domain.displayName for domain in (ctrl.newAccountOptions.availableDomains | orderBy:'formattedName') track by domain.name"
-                        >
-                        </select>
-                    </div>
-                    <small
-                        class="help-block"
-                        data-translate="exchange_tab_ALIAS_taken_error_message"
-                        data-ng-if="ctrl.takenEmailError"
-                    ></small>
-                </div>
+        <!--Account info section-->
+        <div class="row oui-box oui-box_light border-0 p-0">
```

Comment:
**issue:** Same comments as `packages/manager/modules/exchange/src/account/add/account-add.html` file.

Reply:
Got your point. I removed the oui-box class.
Changed for p with oui content style.